sheetstorm-deployment/sheetstorm-vm/configure-sheetstorm-vm.yml

```diff
@@ -39,10 +47,24 @@
             owner:  sheetstorm
             group:  sheetstorm
 
+    - become:      yes
+      become_user: vagrant
+      block:
+        - name:    Reset sheetstorm database
+          command: ~/bin/reset-db.sh sheetstorm
+          when:    reset_database|bool
+
         - name: Migrate database and generate static files
           include_role:
             name: ../roles/configure_sheetstorm_environment
           vars:
             update_sheetstorm_application: 'no'
             generate_static_files:         'yes'
             migrate_database:              'yes'
+
+        - name: Load django fixture
+          django_manage:
+            command:    loaddata
+            app_path:   "{{ sheetstorm_dir }}"
+            fixtures:   "{{ django_fixtures_dir }}/site.yaml"
+            virtualenv: "{{ sheetstorm_virtualenv_dir }}"
```

Comment:
Please also add fixture `employees/fixtures/task_activities.yaml`. This two fixtures will be only loaded when database will be also migrated?

Reply:
Done. I added second fixture. The fixtures will be loaded only after the migration process is completed.